lib/spack/spack/platforms/_base_classes.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

Comment:
call this platform.py then?

Reply:
Done in 0ebda4d753f6e7f7570c98ae2314a63d959237c6